File: java/PrintVowelAndConsonent.java
package com;

import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class PrintVowelAndConsonent {
	public static void main(String[] args) {
		String sentance = "The quick brown fox jumps over the lazy dog.";
		ArrayList<Character> vowels = new ArrayList<>();
		ArrayList<Character> consonent = new ArrayList<>();
		char[] stringArray = sentance.toCharArray();
		for (int i = 0; i < sentance.length(); i++) {
			char ch = sentance.charAt(i);
			if(isVowel(ch)) {
				vowels.add(ch);
			}else if(ch!=' ') {
				consonent.add(ch);
			}
		}
		
		for(char x:vowels) {
			System.out.println(x);
		}
		for(char x:consonent) {
			System.out.println(x);
		}
	}

	public static boolean isVowel(char ch) {
		return ch == 'a' || ch == 'e' || ch == 'i' || ch == 'o' || ch == 'u';
	}
}

File: javascript/PrintVowelAndConsonent.js
'use strict';


let s = "The quick brown fox"
let v=[];
let c=[];

function vowelsAndConsonants(s) {
    
    // console.log(s.length)
    for(let i=0;i<s.length;i++){
        if(checkVowel(s[i])){
           
            v.push(s[i])
           
        }else if(s[i]!==''){
           
            c.push(s[i])
        }
    }
    // console.log(v)
    // console.log(c)
    
    for(let i=0;i<v.length;i++){
        console.log(v[i])
    }
    
    for(let i=0;i<c.length;i++){
        console.log(c[i])
    }
}

function checkVowel(ch){
    return  (ch === 'a' || ch ==='e' || ch === 'i' || ch === 'o' || ch === 'u');
}

vowelsAndConsonants(s);
